engine.py

```diff
@@ -351,7 +444,7 @@ def __str__(self) -> str:
 
 class PicovoiceLeopardEngine(Engine):
     def __init__(self, access_key: str):
-        self._leopard = pvleopard.create(access_key=access_key)
+        self._leopard = pvleopard.create(access_key=access_key, enable_diarization=False)
```

Comment:
I think this is false by default

Reply:
I think so, I thought it's good to be explicit here

Reply:
I think it creates a bit of confusion. This communicates to the reader that it's enabled by default.

Reply:
okay, removed it